Automattic/gridicons
# Build: update svg-to-pdfkit to 0.1.8 (exact)

Checking the problem raised in #309 it seems that for some reason the PDF build system broke again (past, see #258). I wasn't able to pinpoint the issue (maybe some specific package?), so the workaround as before is to pick an updated version, and reset the PDF generation.

This PR thus:

* Updates `svg-to-pdfkit` ([npm](https://www.npmjs.com/package/svg-to-pdfkit)) to 0.1.8
* Updates `pdfkit` ([npm](https://www.npmjs.com/package/pdfkit)) to 0.11.0
* Resets the PDF generation

Testing:

1. Try to do a clean pull (or delete `node_modules`) and check that on rebuild no files are marked as changed
2. Run `grunt` and make sure again no files got changed
3. Try to change any SVG source, and check that the files change as expected (git marks them as modified)
4. Revert that SVG change, and check that the files are now rebuilt back (i.e. no more git modified files)
5. Review that the `story` icon and the `stats-alt-2` icon are the same as `master` branch
